filecoin-proofs/src/constants.rs

```diff
@@ -128,6 +128,13 @@ pub type DefaultBinaryTree = storage_proofs::merkle::BinaryMerkleTree<DefaultTre
 pub type DefaultOctTree = storage_proofs::merkle::OctMerkleTree<DefaultTreeHasher>;
 pub type DefaultOctLCTree = storage_proofs::merkle::OctLCMerkleTree<DefaultTreeHasher>;
 
+// Generic shapes
+pub type SectorShapeBase = LCTree<DefaultTreeHasher, U8, U0, U0>;
+pub type SectorShapeSub2 = LCTree<DefaultTreeHasher, U8, U2, U0>;
+pub type SectorShapeSub8 = LCTree<DefaultTreeHasher, U8, U8, U0>;
+pub type SectorShapeTop2 = LCTree<DefaultTreeHasher, U8, U8, U2>;
+
```

Comment:
The one further step I would take is to use these types in the definitions directly below. That will also make the correctness of the following predicates easiest to verify and keep in sync.